policy/modules/services/xserver.te

```diff
@@ -968,6 +968,8 @@ optional_policy(`
     dbus_session_bus_client(xdm_t)
     dbus_connect_session_bus(xdm_t)    
 
+    dbus_watch_config_dirs(xdm_t)
```

Comment:
This has been addressed in commit 8ad8478535f5daf40fefd00833240adf15275d39 with a transition.